app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -0,0 +1,59 @@
+module ProviderInterface
+  class DecisionsController < ProviderInterfaceController
+    before_action :set_application_choice_and_presenter
+
+    def respond; end
+
+    def submit_response
+      Rails.logger.warn params.inspect
+      decision = params[:application_choice][:decision] if params[:application_choice]
+      if decision == 'offer'
+        redirect_to action: :new_offer
+      elsif decision == 'reject'
+        redirect_to action: :new_reject
+      else
+        raise 'Unexpected application_choice decision'
```

Comment:
Should there be some handling if a provider clicks continue on the `/provider/applications/{id}/respond` page without first selecting an option?

Currently we just get an exception.

Reply:
No, this code is actually part of ```#submit_response```, which you only get to via a POST. You'd have to post to this url without specifying 'offer' or 'reject' to get the error.

Reply:
Yes but when you first navigate to the `respond` page neither option is selected. 

<img width="1087" alt="Screenshot 2019-11-15 at 11 56 01" src="https://user-images.githubusercontent.com/47318392/68941924-12832300-079f-11ea-91fe-b58137356a6c.png">

And when you click continue without selecting an option you get an exception.

<img width="1332" alt="Screenshot 2019-11-15 at 11 57 54" src="https://user-images.githubusercontent.com/47318392/68941961-2f1f5b00-079f-11ea-8b98-ac622725f60d.png">

Reply:
Oh I see. I'll remove the exception for now and return no content. Once we have a warning flash component I'll add a warning to the page.